examples/htool_payload.h

```diff
@@ -0,0 +1,48 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef LIBHOTH_EXAMPLES_HTOOL_PAYLOAD_H_
+#define LIBHOTH_EXAMPLES_HTOOL_PAYLOAD_H_
+
+#include <stdint.h>
+
+struct payload_status_response_header {
```

Comment:
Should these two structure definitions be in host_commands.h instead?

Reply:
Yes this sounds reasonable, since `payload_status_response_header` is directly bit-cast from the response buffer, similarly to other structs in host_commands.h. So moved to `host_commands.h`.